.changeset/mighty-cows-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
```

Comment:
[note] This fix will increase Stylelint errors, so "minor" is correct according to our versioning policy.

https://github.com/stylelint/stylelint/blob/a5965a09b557fd4a5420a6b48f1c1353ecf83aa2/docs/about/semantic-versioning.md?plain=1#L17-L21